scss/bitstyles/atoms/badge/badge.stories.mdx

```diff
@@ -1,117 +1,133 @@
-```
+Badges use the theming system for their color & background-color properties — take a look at [themes](/docs/design-tokens-themes--page) for details on how to customize your themes.
+
+The custom buttons that are available in each badge are implemented in the Button component — they’re component-level additions to each theme, just for the default button color-variant. See the [Button component](/docs/atoms-button-colors--page) for details on how to customize their appearance.
 
-The keys from those colors that will be used are specified with the `$bitstyles-badge-color` and `$bitstyles-badge-background-color`, and are applied to all variants. With the following settings, the `gray` variant will have color `gray` value `80`, with background color `gray` value `10`. The `danger` variant will have color `danger` with value `80` and background-color `danger` at value `10` etc.
+The badge shadows are component-level additions to the theme, which can be specified with `$theme-variants`. Pass the name of the variant and a Sass map for each variant:
 
 ```scss
-$color: '80';
-$background-color: '10';
+@use '~bitstyles/scss/bitstyles/tools/shadows';
+@use '~bitstyles/scss/bitstyles/atoms/badge' with (
+  $theme-variants: (
+    'theme-name': (
+      'shadow': shadows.get('brand-2'),
+    ),
+  )
+);
 ```
 
-Padding can be specified with `$bitstyles-badge-padding-horizontal` and ``$bitstyles-badge-padding-vertical`:
+### Size variants
+
+To configure your size variants override the `$size-variants` variable, with each breakpoint (use `setup.$no-media-query` for the mobile-first base styles) at the root, containing a size-variant name and Sass map with the sizing properties:
 
 ```scss
-$padding-vertical: size.get('s6');
-$padding-horizontal: size.get('m');
+@use '~bitstyles/scss/bitstyles/tools/size';
+@use '~bitstyles/scss/bitstyles/atoms/badge' with (
+  $size-variants: (
+    '#{setup.$no-media-query}': (
+      '': (
+        'padding-vertical': size.get('s6'),
+        'padding-horizontal': size.get('l1'),
+        'gap': size.get('s6'),
+        'extra-spacing': 0rem,
+      ),
+    ),
+    'l': (
+      '': (
+        'padding-vertical': size.get('s4'),
+        'padding-horizontal': size.get('l3'),
+        'gap': size.get('s1'),
+        'extra-spacing': size.get('s4'),
+      ),
+      'small': (
+        'padding-vertical': size.get('s7'),
+        'padding-horizontal': size.get('s7'),
+        'gap': size.get('s6'),
+        'extra-spacing': 0rem,
+      ),
+    ),
+  )
+);
 ```
+
+The size-related properties available for configuration are:
+
+| Property name        | Description                                                                                 |
+| -------------------- | ------------------------------------------------------------------------------------------- |
+| `padding-vertical`   | `padding-top` and `padding-bottom` properties                                               |
```

Comment:
Any reasons why the words "vertical" and "horizontal" were preferred to the more traditional "x" and "y"?

Reply:
Yeh that’s a great point 🤔 

`x`/`y` are used for the padding & margin utility classes, but the sass variables for components have always been named with `horizontal`/`vertical`. I think there’s no good reason for that difference though, and actually I’ve been wondering when it might make sense to switch to using `padding-block`, `margin-block`, `padding-inline` etc for these. Caniuse shows two years of support for these logical directions (https://caniuse.com/css-logical-props)

I’ll keep this naming for now, and add an issue to investigate switching to using the logical properties (and the matching naming) everywhere